dataactcore/aws/sqsHandler.py

```diff
@@ -28,6 +38,31 @@ def purge():
         sess.query(SQS).delete()
         sess.commit()
 
+    @property
+    def attributes(self):
+        mock_redrive = '{{"deadLetterTargetArn": "FAKE_ARN:{}", ' \
+                       '"maxReceiveCount": {}}}'.format(self.UNITTEST_MOCK_DEAD_LETTER_QUEUE, self.max_receive_count)
+        return {"ReceiveMessageWaitTimeSeconds": "10", "RedrivePolicy": mock_redrive}
+
+
+class SQSMockDeadLetterQueue:
+    logger = logging.getLogger(__name__)
+
+    @staticmethod
+    def send_message(MessageBody, MessageAttributes=None):  # noqa
+        SQSMockDeadLetterQueue.logger.debug("executing SQSMockDeadLetterQueue.send_message({}, {})".format(
+            MessageBody, MessageAttributes))
+        return {"ResponseMetadata": {"HTTPStatusCode": 200}}
+
+    @staticmethod
+    def receive_messages(WaitTimeSeconds, AttributeNames=None, MessageAttributeNames=None,  # noqa
+                         VisibilityTimeout=30, MaxNumberOfMessages=1):  # noqa
+        SQSMockDeadLetterQueue.logger.debug("executing SQSMockDeadLetterQueue.receive_messages("
```

Comment:
This has a purpose and should not be removed.
Since we only have the one "mock queue" (which is the `SQS` database table) in play during local development and unit tests, this makes up for us not having another dead letter queue. We don't exactly need one, since the dead letter queue is only there to give some history of messages that "died". This log statement basically becomes the historical record of a message that went to the _"mock"_ dead letter queue. 

So if a user developing locally ran into a scenario where their message "died", they could see a track record of this in the logs. 

Similarly it validates dead letter routing during unit tests.